tests/stockfish/test_models.py

```diff
@@ -87,8 +87,8 @@ def test_set_fen_position_starts_new_game(self, stockfish):
         stockfish.set_fen_position("3kn3/p5rp/1p3p2/3B4/3P1P2/2P5/1P3K2/8 w - - 0 53")
         assert stockfish.info == ""
 
-    def test_set_fen_position_second_argument(self):
-        stockfish = Stockfish(depth=16)
+    def test_set_fen_position_second_argument(self, stockfish):
+        stockfish.set_depth(16)
```

Comment:
Many other unit tests were failing because new instances of the stockfish class were trying to be called. All the unit tests should run with a single stockfish instance.